src/redist/targets/BundledManifests.targets

```diff
@@ -68,13 +68,15 @@
     </GenerateGuidFromName>
       
     <ItemGroup>
-      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\**\*"
+      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\*"
                        Condition="Exists('%(RestoredNupkgContentPath)\data')"
-                       DestinationPath="$([MSBuild]::ValueOrDefault('%(WorkloadManifestId)', '').ToLower())/"/>
-
-      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\**\*.json;%(BundledManifests.RestoredNupkgContentPath)\**\*.targets"
-                       Condition="!Exists('%(BundledManifests.RestoredNupkgContentPath)\data')"
-                       DestinationPath="$([MSBuild]::ValueOrDefault('%(BundledManifests.WorkloadManifestId)', '').ToLower())/"/>
+                       DestinationPath="$([MSBuild]::ValueOrDefault('%(WorkloadManifestId)', '').ToLower())"
+                       RestoredNupkgContentPath="%(RestoredNupkgContentPath)"
+                       WorkloadManifestId="%(WorkloadManifestId)"/>
+                       
+      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\localize\*"
```

Comment:
Would it be better to try to do this with `%(RecursiveDir)` instead of having separate includes?

Reply:
I often have trouble too when I try to use it.  I would look for other uses of `RecursiveDir` in the installer repo build.  Also, here's an example of it where it's used in the product: https://github.com/dotnet/sdk/blob/5828663294bb4d048a02892e4efcf11d1f0c5bbe/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L90

I wouldn't spend too much effort on it if you don't want to though.

Reply:
Yeah that looks similar to the way I was trying to use it... I'm going to go ahead and go in with the current change, since we don't expect any other subdirs and this logic will be removed in 6.0.2xx anyways.